usecases/traverser/explorer.go

```diff
@@ -282,6 +282,13 @@ func (e *Explorer) Hybrid(ctx context.Context, params dto.GetParams) ([]search.R
 		return nil, err
 	}
 
+	if params.Pagination.Limit > 0 && params.Pagination.Offset >= 0 && len(res) > params.Pagination.Offset {
+		if len(res) > params.Pagination.Limit+params.Pagination.Offset {
+			res = res[:params.Pagination.Limit+params.Pagination.Offset]
+		}
+		res = res[params.Pagination.Offset:]
+	}
```

Comment:
shouldn't there be one more case handled?
when `len(res) <= params.Pagination.Offset` empty slice should be returned?